### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by arpit01923**
Created a simple basic modal component
image

### CODE REVIEW

File: css/example/Modal.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../components/Modal.css">
+    <script src="https://kit.fontawesome.com/dcbd873a92.js" crossorigin="anonymous"></script>
```
Comment (Naveenchandar) [inline css/example/Modal.html]: To use font awesome scripts are not needed. Can be handled with link. Please look for font awesome cdn

File: css/components/Modal.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    display: flex;
+    justify-content: space-between;
+}
+
+.content{
+    padding: 1rem;
+}
+
+.btn{
+    display: flex;
```
Comment (Naveenchandar) [inline css/components/Modal.css]: If flex property is used more often, You can move this to utility file.

File: css/example/Modal.html (hunk 1 above)
Comment (Arpit-dotcom) [reply css/example/Modal.html:9]: how to use it

Comment (arpit01923) [reply css/example/Modal.html]: done

**Review by ckahub — COMMENTED** [review]
looks good to me

**Comment by ckahub** [comment]
you can go ahead and merge file

File: css/components/Modal.css (hunk 2 above)
Comment (arpit01923) [reply css/components/Modal.css]: done

**Pull request merged**